src/main/java/org/cef/browser/CefRenderer.java

```diff
@@ -8,6 +8,10 @@
 
```

Comment:
everything in this file needs a careful look, I did a LOT of copying+pasting here and there may be some junk in here that should go.  renders seemingly OK though